### PR TITLE
chore: check tx exists by address in tx service

### PR DESCRIPTION
**Pull request by classicalliu**

### CODE REVIEW

File: packages/neuron-wallet/src/services/transactions.ts [hunk 1]
```diff
@@ -69,6 +69,11 @@ export default class TransactionsService {
     return transactions.find(tx => tx.hash === hash)
   }
 
+  // check weather the address has history transactions
```
Comment (Keith-CY) [inline packages/neuron-wallet/src/services/transactions.ts]: `wether` not `weather`

Comment (classicalliu) [reply packages/neuron-wallet/src/services/transactions.ts]: ...OK

File: packages/neuron-wallet/src/services/transactions.ts [hunk 2]
```diff
@@ -69,6 +69,11 @@ export default class TransactionsService {
     return transactions.find(tx => tx.hash === hash)
   }
 
+  // check whether the address has history transactions
+  public static isExist = (_address: string): boolean => {
```
Comment (ashchan) [inline packages/neuron-wallet/src/services/transactions.ts]: `isExist` is incorrect in English grammar.

File: packages/neuron-wallet/src/services/transactions.ts [hunk 3]
```diff
@@ -69,6 +69,11 @@ export default class TransactionsService {
...
+    return Math.random() > 0.5
```
Comment (ashchan) [inline packages/neuron-wallet/src/services/transactions.ts]: `>= 0.5` gives us 50% probability, while `> 0.5` doesn't.

File: packages/neuron-wallet/src/services/transactions.ts (hunk 2 above)
Comment (Keith-CY) [reply packages/neuron-wallet/src/services/transactions.ts]: the meaning of “isExist address” is incorrect, use “hasTransactions” instead

**Comment by classicalliu** [comment]
@Keith-CY call for review

**Pull request merged**